bigframes/dtypes.py

```diff
@@ -876,3 +882,6 @@ def lcd_type_or_throw(dtype1: Dtype, dtype2: Dtype) -> Dtype:
     "STRING",
     "ARRAY",
 }
+
+
+TIMEDELTA_DESCRIPTION_TAG = "#microsecond"
```

Comment:
Very much a nit, but I think "microseconds" sounds better for a description of timedelta.

```suggestion
TIMEDELTA_DESCRIPTION_TAG = "#microseconds"
```

Alternatively, do we want to align with the JSON format used for remote functions descriptions? I think there are benefits to this implementation, but it'd probably be good to document that choice in the design if we haven't already.

Reply:
Could you share more about this JSON format description?